.github/actions/rosa-cleanup-clusters/action.yml

```diff
@@ -8,16 +8,96 @@ inputs:
     tf-bucket:
         description: Bucket containing the clusters states
         required: true
+
     tf-bucket-region:
         description: Region of the bucket containing the resources states, if not set, will fallback on AWS_REGION
+
     max-age-hours-cluster:
         description: Maximum age of clusters in hours
         required: false
         default: '20'
 
+    rosa-cli-version:
+        description: Version of the ROSA CLI to use
+        default: latest
+
+    # inherited from https://github.com/hashicorp/setup-terraform/blob/main/action.yml
+    tf-cli-config-credentials-hostname:
+        description: |
+            The hostname of a HCP Terraform/Terraform Enterprise instance to place within the credentials block of the Terraform CLI configuration
+            file. Defaults to `app.terraform.io`.
+        default: app.terraform.io
+        required: false
+
+    tf-cli-config-credentials-token:
+        description: |
+            The API token for a HCP Terraform/Terraform Enterprise instance to place within the credentials block of the Terraform CLI configuration
+            file.
+        required: false
+
+    tf-terraform-version:
+        description: |
+            The version of Terraform CLI to install. Instead of full version string you can also specify constraint string starting with "<" (for
+            example `<1.13.0`) to install the latest version satisfying the constraint. A value of `latest` will install the latest version of Terraform
+            CLI. Defaults to `latest`.
+        default: latest
+        required: false
+
+    tf-terraform-wrapper:
+        description: |
+            Whether or not to install a wrapper to wrap subsequent calls of the `terraform` binary and expose its STDOUT, STDERR, and exit code
+            as outputs named `stdout`, `stderr`, and `exitcode` respectively. Defaults to `true`.
+        default: 'true'
+        required: false
+
+    openshift-version:
+        description: Version of the OpenShift to install
+        required: true
+        # renovate: datasource=custom.rosa-camunda depName=red-hat-openshift versioning=semver
+        default: 4.17.10
+
+    awscli-version:
+        description: Version of the aws cli to use
+        required: true
```

Comment:
for both of them.
is it required if it has a default?

Reply:
yep, and back then already mentioned the contradiction ^^, that's why I approved it regardless.
Doesn't change anything functionality wise, but also doesn't require it.